book/src/concepts/nested_objects.md

```diff
@@ -24,7 +24,7 @@ For use as a parameter in an invokable write `secondary_object: &mut crate::mymo
 The following example shows a nested object as a property and parameter.
 
 ```rust,ignore,noplayground
-{{#include ../../../examples/qml_features/src/nested.rs:book_macro_code}}
+#include ../../../examples/qml_features/rust/src/nested.rs:book_macro_code
```

Comment:
I removed the `{{`/`}}` to get `mdbook build` to run without errors.

Reply:
Interesting, that was in a commented block before as nested objects have been removed for now. But this is OK for now ..